extensions/sql-database-projects/src/models/deploy/deployService.ts

```diff
@@ -0,0 +1,403 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppSettingType, IDeployProfile, ILocalDbSetting } from './deployProfile';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+import { Project } from '../project';
+import * as constants from '../../common/constants';
+import * as utils from '../../common/utils';
+let fse = require('fs-extra');
+let path = require('path');
+import * as childProcess from 'child_process';
+import * as vscode from 'vscode';
+import * as os from 'os';
+import { ConnectionResult } from 'azdata';
+import * as templates from '../../templates/templates';
+
+interface ValidationResult {
+	errorMessage: string;
+	validated: boolean
+}
+export class DeployService {
+
+	constructor(private _outputChannel: vscode.OutputChannel) {
+	}
+
+	private createConnectionStringTemplate(runtime: string | undefined): string {
+		switch (runtime?.toLocaleLowerCase()) {
+			case 'dotnet':
+				return constants.defaultConnectionStringTemplate;
+				break;
+			// TODO: add connection strings for other languages
+			default:
+				break;
+		}
+		return '';
+	}
+
+	private findAppRuntime(profile: IDeployProfile, appSettingContent: any): string | undefined {
+		switch (profile.appSettingType) {
+			case AppSettingType.AzureFunction:
+				return <string>appSettingContent?.Values['FUNCTIONS_WORKER_RUNTIME'];
+			default:
+		}
+		return undefined;
+	}
+
+	public async updateAppSettings(profile: IDeployProfile): Promise<void> {
+		// Update app settings
+		//
+		if (!profile.appSettingFile) {
+			return;
+		}
+		this.logToOutput(`Updating app setting: ${profile.appSettingFile}`);
+
+		let content = JSON.parse(fse.readFileSync(profile.appSettingFile, 'utf8'));
```

Comment:
Do you handle errors parsing in a clean way?

Reply:
I'll add a task to fix it in another PR